### PR TITLE
4.16.0 release docs

### PR DESCRIPTION
**Pull request by nvazquez**

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
requesting docbuild

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Review by GabrielBrascher — APPROVED** [review]
Thanks for the 4.16.0 release documentation upgrades @nvazquez!

Changes LGTM based on:
1. quick evaluation on small key points in the code (that's generated by script, no reason to look at the whole code).
2. checked the builded documentation

**Comment by rhtyd** [comment]
@blueorangutan docbuild

**Comment by rhtyd** [comment]
@blueorangutan docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,6 +17,33 @@
...
+• Script cloudstack-setup-databases improvement
+• Add mac learning mode in network offering
+• API-call to declare host as Degraded
+• New API endpoint to update pod management network IP range
+• Add New API endpoint: UpdateVlanIpRange
+
+The full list of new features can be found in the project release notes at
+http://docs.cloudstack.apache.org/en/4.16.0.0/releasenotes/changes.html
+
+What's New in 4.15.2.0
```
Comment (rhtyd) [inline source/releasenotes/about.rst:44]: This needs to go away

**Comment by rhtyd** [comment]
@blueorangutan docbuild

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Comment by nvazquez** [comment]
@blueorangutan docbuild

**Comment by rhtyd** [comment]
LGTM - merging this as can't collaborate over nicolas's private PR/fork.

**Pull request merged**